airflow-core/src/airflow/api_fastapi/common/parameters.py

```diff
@@ -540,6 +540,22 @@ def is_active(self) -> bool:
             self.value.lower_bound is not None or self.value.upper_bound is not None
         )
 
+class _IsDagScheduledFilter(BaseParam[bool]):
+    """Filter on timetable_description."""
+
+    def to_orm(self, select: Select) -> Select:
+        if not self.value is None and self.skip_none:
+            if not self.value:
+                return select.where(DagModel.timetable_description.ilike("%Never%"))
+            else:
+                return select.where(DagModel.timetable_description.not_like("%Never%"))
```

Comment:
A small speedup for query 

```suggestion
                return select.where(DagModel.timetable_description.ilike("Never%"))
            else:
                return select.where(DagModel.timetable_description.not_like("Never%"))
```

Reply:
Hi @jason810496 ,

have implemented suggested changes, 

please check.